spec/models/events/creator_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Events::Creator, :type => :model do
+
+  before do
+    Timecop.freeze(Time.local(1990))
```

Comment:
One approach to handle the fail with this is to do `be_within(1).of(start_date)` in the testing spots. Most likely your DB has a different float precision if that was failing